lib/phoenix/socket.ex

```diff
@@ -520,6 +546,10 @@ defmodule Phoenix.Socket do
     :ok
   end
 
+  @doc false
+  def __log__(Phoenix.LiveReloader.Socket, _opts), do: false
```

Comment:
We shouldn't hardcode this here. Instead we should set log: false in the live reloader lib and ship a new version.

Reply:
Agree, which we will do, but folks who don't `mix deps.update phoenix_live_reload` will get spammy logs. It's not ideal to match on this here, but I think we do something similar elsewhere for ergonomics. Thoughts?

Reply:
Then they can update their `phoenix_live_reload`. :)

Reply:
Fair :D